docs/cluster-management/developing-locally.md

```diff
@@ -248,39 +250,38 @@ git clone git@github.com:screwdriver-cd/queue-worker.git
     # Externally routable URL for the Artifact Store
     store: http://$YOUR_IP:9002
     
- redis:
-    # Host of redis cluster
-    host: 127.0.0.1
-    # Password to connect to redis cluster, if set in first step
-    password: 'c1cd-$cr3wdriver-cd'
-    # Port of redis cluster
-    port: 6379
-    # Prefix for the queue name if needed
-    # prefix: 'beta-'
-    # Flag to enable client for TLS-based communication
-    tls: false
+ queue:
+    # Configuration of the redis instance containing resque
+    redisConnection:
+        host: "127.0.0.1"
+        port: 6379
+        options:
+            # Set password if set in previous step
+            password: 'c1cd-$cr3wdriver-cd'
+            tls: false
+        database: 0
+        prefix: ""
 ```
 
-### Step 3: Modify screwdriver/config/local.yaml and change executor configuration
+### Step 3: Modify screwdriver/config/local.yaml, change executor configuration and add queue uri 
 
 ```yaml
-executor:
+ ecosystem:
+    # Externally routable URL for the User Interface
+    ui: http://sd.screwdriver.cd:4200
+    # Externally routable URL for the API
+    api: http://$YOUR_IP:9001
+    # Externally routable URL for the Artifact Store
+    store: http://$YOUR_IP:9002
+    # Routable URI of the queue service
+    queue: http://$YOUR_IP:9003
```

Comment:
where is this port configured? is it default somewhere?

Reply:
> where is this port configured? is it default somewhere?

Yes, this is default in queue-service.